packages/calypso-e2e/src/lib/pages/editor-page.ts

```diff
@@ -120,46 +120,23 @@ export class EditorPage {
 	 * @returns {Promise<Frame>} iframe holding the editor.
 	 */
 	async waitUntilLoaded(): Promise< void > {
-		// Once https://github.com/Automattic/wp-calypso/issues/57660 is resolved,
-		// the next line should be removed.
-		const editor = await this.getEditorHandle();
-
-		const titleLocator = editor.locator( selectors.editorTitle );
-		await titleLocator.waitFor( { timeout: EXTENDED_TIMEOUT } );
```

Comment:
This is a moderately large change.

Instead of waiting for the Title block in the editor to appear, I have made the calculated decision to wait for a request to finish loading.

I feel confident that this will not lead to intermittent failures:
1. if the request and response to the`posts` endpoint has not completed within 30s, it is unlikely that waiting additional time for the editor title to appear would have been any safer.
2. waiting for the editor title causes issues with Gutenberg 14.9.1 where an iframe now wraps the editor canvas. 

For the second point, while there is a follow-up PR to address the inner iframe, I decided that relying less on UI elements and more on the network requests would ultimately result in a more reliable loading pattern.

I'm open to hearing opinions on this change. If required, it is possible to abandon this change and look forward to a fix in the follow-up PR.